l10n_ch_delivery_carrier_label_quickpac/data/product.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo noupdate="1">
+    <record id="product_quickpac_service" model="product.product">
+        <field name="image" type="base64" file="delivery_carrier_label_quickpac/static/img/quickpac-logo.jpg"/>
```

Comment:
```suggestion
        <field name="image" type="base64" file="l10n_ch_delivery_carrier_label_quickpac/static/img/quickpac-logo.jpg"/>
```

Reply:
same fix to be applied in all places where the logo is referenced

Reply:
Already done. Travis is restarted ^^